modes/basic-test-mode/src/moreTools.ts

```diff
@@ -208,21 +203,21 @@ const moreTools = [
               context: 'CORNERSTONE',
             },
           ],
-          'Rectangle'
-        ),
-        ToolbarService._createActionButton(
-          'TagBrowser',
-          'list-bullets',
-          'Dicom Tag Browser',
-          [
+          evaluate: 'evaluate.cornerstoneTool',
+        }),
+        ToolbarService.createButton({
+          id: 'TagBrowser',
+          icon: 'list-bullets',
+          label: 'Dicom Tag Browser',
+          tooltip: 'Dicom Tag Browser',
+          commands: [
```

Comment:
This is one place where you need a commands list specific to the tool

Reply:
However, you can use a single commands item instead of an array - eg commands: { commandName: 'openDICOMTagViewer' } - actually, we should let that default to just the string name of a command.